src/citrine/informatics/processors.py

```diff
@@ -182,22 +182,25 @@ class MonteCarloProcessor(Resource['GridProcessor'], Processor, AIResourceMetada
     _resource_type = ResourceTypeEnum.MODULE
 
     max_candidates = properties.Optional(properties.Integer, 'config.max_candidates')
+    mode = properties.Optional(properties.String(), 'config.mode')
 
     typ = properties.String('config.type', default='ContinuousSearch', deserializable=False)
     module_type = properties.String('module_type', default='PROCESSOR')
 
     def _attrs(self) -> List[str]:
-        return ["name", "description", "typ"]
+        return ["name", "description", "mode", "typ"]
 
     def __init__(self,
                  name: str,
                  description: str,
                  max_candidates: Optional[int] = None,
+                 mode: Optional[str] = None,
```

Comment:
What can we document about this parameter?

Reply:
Annotated to let people know not to worry about it.